uint8 motor_id

float32 position_estimate
float32 velocity_estimate
float32 torque_estimate